src/Model/Worker.py

```diff
@@ -33,7 +35,24 @@ def __init__(self, fn, *args, **kwargs):
         self.kwargs = kwargs
         self.signals = WorkerSignals()
 
-        self.kwargs['progress_callback'] = self.signals.progress
+        # When the constructor for this class is called with a keyword argument 'progress_callback' set to True, this
```

Comment:
Is there an automated test for this class?
Is it feasible to put together a small test (even calculating pi to ten decimal places would be fine), and maybe try it three different ways ( progress_callback is True, progress_callback is False, and no progress_callback kwarg parameter provided)?

Let's aim to add a minimal test for every module touched that doesn't have a test.
And if the module has a test, update or add to the test so the change itself is tested.